build/csslint/xfail.list

```diff
@@ -53,6 +53,7 @@ apps/settings/style/downloads.css 0 3
 apps/sms/style/root.css 0 1
 apps/sms/style/sms.css 0 4
 apps/system/mobile_id/style/mobile_id.css 0 16
+apps/system/style/devtools/devtools_auth.css 1 0
```

Comment:
The line `object-fit: cover;` gave a linting error, but it definitely has an effect.

Is there somewhere I can tell the linter "no, really, this does work!"? Wasn't sure how, so I thought I should just accept the error here.